enginetest/queries/variable_queries.go

```diff
@@ -60,6 +61,41 @@ var VariableQueries = []ScriptTest{
 			{uint64(2)},
 		},
 	},
+	{
+		Name: "variable scope is included in returned column name when explicitly provided",
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:    "select @@max_allowed_packet;",
+				Expected: []sql.Row{{1073741824}},
+				ExpectedColumns: sql.Schema{
+					{
+						Name: "@@max_allowed_packet",
+						Type: types.Uint64,
+					},
+				},
+			},
+			{
+				Query:    "select @@session.max_allowed_packet;",
+				Expected: []sql.Row{{1073741824}},
+				ExpectedColumns: sql.Schema{
+					{
+						Name: "@@SESSION.max_allowed_packet",
```

Comment:
Could mean we need to add an alias in the select exprs, similar to what we do with other expressions today.

Reply:
Not including the scope is enough to fix the immediate customer issue, but the casing difference was sticking out to me, too. Instead of returning a boolean value for whether the scope was explicitly specified or not, we could return the explicitly specified scope and then we'd have the original casing that we could preserve.